src/main/java/net/minecraftforge/client/EnumHelperClient.java

```diff
@@ -29,49 +29,95 @@
 
 import javax.annotation.Nullable;
 
-public class EnumHelperClient extends EnumHelper
-{
-    private static Class<?>[][] clientTypes =
-    {
+/**
```

Comment:
`EnumHelperClient` wont be needed, no need for javadocs.

Reply:
Done